### PR TITLE
test_pylint fallback its path on current working directory if none de…

### PR DESCRIPTION
**Pull request by jcdrubay**
…fined as option or environment variable TRAVIS_BUILD_DIR

### CODE REVIEW

File: travis/run_pylint.py [hunk 1]
```diff
@@ -3,13 +3,14 @@
 
 from __future__ import print_function
 
-import os
-import sys
-
 import click
+import os
 import pylint.lint
+import sys
```
Comment (dreispt) [inline travis/run_pylint.py]: Imports from the standard library should come before imports from third party packages.
See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

**Comment by jcdrubay** [comment]
I took into account the small comment from @dreispt and created #397 rebasing this PR to have a clean commit history.

**Comment by dreispt** [comment]
You don't need to create a new PR.
Just add your commit to this branch and push it, and this PR will be updated.
Github allows us to squash the commits when merging, so you don't need to worry about that.

In case you care and do want to tidy your commit history, you can use `git commit --amend` to modify the last commit, instead of adding a new one.

File: travis/run_pylint.py [hunk 2]
```diff
@@ -78,7 +80,7 @@ def run_pylint(paths, cfg, beta_msgs=None, sys_paths=None, extra_params=None):
 
 @click.command()
 @click.option('paths', '--path', envvar='TRAVIS_BUILD_DIR',
-              multiple=True, type=CLICK_DIR, required=True,
+              multiple=True, type=CLICK_DIR,
```
Comment (moylop260) [inline travis/run_pylint.py]: What about ` default=os.getcwd()`?

**Comment by dreispt** [comment]
LGTM

File: travis/run_pylint.py [hunk 3]
```diff
@@ -10,6 +10,8 @@
 import pylint.lint
 
 import getaddons
+import travis_helpers
```
Comment (moylop260) [inline travis/run_pylint.py]: Please, fix `./travis/run_pylint.py:13:1: F401 'travis_helpers' imported but unused`
https://travis-ci.org/OCA/maintainer-quality-tools/jobs/183560210#L468

File: travis/run_pylint.py [hunk 4]
```diff
@@ -97,6 +100,7 @@ def main(paths, config_file, msgs_no_count=None,
     to check fails of odoo modules.
     If expected errors is equal to count fails found then
     this program exit with zero otherwise exit with counted fails"""
+
```
Comment (moylop260) [inline travis/run_pylint.py]: If this change is not required then remove it

File: travis/run_pylint.py [hunk 5]
```diff
@@ -78,7 +80,8 @@ def run_pylint(paths, cfg, beta_msgs=None, sys_paths=None, extra_params=None):
 
 @click.command()
 @click.option('paths', '--path', envvar='TRAVIS_BUILD_DIR',
-              multiple=True, type=CLICK_DIR, required=True,
```
Comment (moylop260) [inline travis/run_pylint.py]: Could you use `required=True`?

**Comment by jcdrubay** [comment]
Thanks @moylop260 and @dreispt for your help. :)

**Pull request merged**